eoscostarica/proton-affiliate
# fix: outdated status for referral payments

### fix: outdated status for referral payments

### What does this PR do?

- Resolve #128
- Resolve #113
- Resolve #121 

### Steps to test

1. make run
2. login with a referrer user or admin user
3. check the table items 

#### CheckList
